### PR TITLE
blog post for run on login

### PR DESCRIPTION
**Pull request by tropicadri**
Adding the post for run on login, it was originally proposed for web.dev but so far it has only been implemented in chromium browsers.

### CODE REVIEW

File: site/en/blog/run-on-login/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+platform specific applications. Now it is available for installed web applications too.
+
+## Save users time
+
+Users want to automatically run a set of applications they always use. A
+typical example is high-engagement applications like email clients, chat apps,
+monitoring dashboards, and real-time data display apps, to name just a few.
+
+Auto-starting these apps at login streamlines the user experience by avoiding
+having to find these apps to launch in menus.
```
Comment (jpmedley) [inline site/en/blog/run-on-login/index.md:26]:
```suggestion
Auto-starting these apps at login streamlines the user experience by saving
users from having to find these apps to launch in menus.
```

Comment (jpmedley) [reply site/en/blog/run-on-login/index.md:26]: Grammatically, the two parts of the sentence are structured as though they refer to the same actor. This change corrects that.

**Review by jpmedley — APPROVED** [review]
@tropicadri Thank you for your patience with the move request. I've suggested two non-critical changes, but have approved the PR to speed things up.

**Comment by tropicadri** [comment]
Thank you! @jpmedley

**Pull request merged**